ketohub/spiders/__init__.py

```diff
@@ -0,0 +1,4 @@
+# This package will contain the spiders of your Scrapy project
+#
+# Please refer to the documentation for information on how to create and manage
+# your spiders.
```

Comment:
Let's just leave this as an empty file. I'd never think to look in an `__init__.py` for folder documentation.